app/Schema/Mysql.php

```diff
@@ -5,7 +5,22 @@
 use PDO;
 use Kanboard\Core\Security;
 
-const VERSION = 93;
+const VERSION = 94;
+
+function version_94(PDO $pdo)
+{
+    $pdo->exec("
+        CREATE TABLE colors (
+            color_id VARCHAR(50) NOT NULL COMMENT 'Backwards compatible format with Kanboard color IDs.',
```

Comment:
I'm not sure if the comment if the schema itself are really necessary. The column names are easily understandable.